canonical/juju-dashboard
# WD-8402 - grant/revoke secrets

## Done

- Add a panel for granting/revoking secrets.

## QA

- Go to the secrets tab for a model with more than one application.
- Open the action menu for a secret.
- Click "Grant"
- Choose an application and save.
- The panel should close and the number of granted counts should update.
- Open the grant panel again and deselect the selected app and select another and save.
- Reopen the panel and the checkboxes should reflect the changes you just made.

## Details

https://warthogs.atlassian.net/browse/WD-8403
https://warthogs.atlassian.net/browse/WD-8402
